### DISCUSSION TITLE
蓝奏云控制台排版混乱

### DISCUSSION THREAD

**Discussion by Bush2021** [F9y4ng/GreasyFork-Scripts#144] [问答专区(Question / Answer)] [answered]
https://pc.woozooo.com/mydisk.php

似乎是最近版本更新后出现的问题?

<img width="960" alt="2022-11-07_135026" src="https://user-images.githubusercontent.com/79072750/200235727-77756d4f-c0e8-4a85-b4ed-ffa12c6bd837.png">

**Comment by F9y4ng** [comment, marked as answer]
木有lanzou的账户，目测应该是字体比例缩放造成的，针对vw、vh的修正因性能及跨域问题已终止，尝试关闭字体缩放（设置为1.000），并使用浏览器缩放替代（ctrl++,ctrl+-）。

**Reply by Bush2021** [reply]
确实是这个原因, 感谢解答！

看来这个功能目前还不太稳定? 我个人蛮喜欢这个功能的, 浏览器全局缩放会把扩展菜单一并放大, 有点不美观, 希望作者能完善！

**Reply by F9y4ng** [reply]
css:zoom本身就属于BLINK内核浏览器的实验性功能，不确定未来的某个版本就会被遗弃掉。

并且，与浏览器端的缩放不同，脚本级缩放由于沙盒内脚本权限的问题有很多不能逾越的限制。很多如同坐标偏移、长度在不同坐标系的修正、样式的兼容等等，有些可以靠脚本修正，有些就显得无能为力。

正如你目前遇到的问题，能不能修正，能，但付出的性能代价（如明显的卡顿、高CPU高内存占用等等）是不能承受的。
